lib/moonshot/commands/new.rb

```diff
@@ -0,0 +1,82 @@
+module Moonshot
+  module Commands
+    class New < Moonshot::Command
+      self.usage = 'new [options]'
+      self.description = 'Creates a new Moonshot project.'
+
+      DEFAULT_DIRECTORY = File.join(__dir__, '..', '..', 'default').freeze
+
+      def execute
+        warn 'Looks like your project is already set up!'
+      end
+
+      class << self
+        def run!(application_name)
+          @application_name = application_name
+
+          create_project_dir
+          copy_defaults
+          create_file(parameter_path)
+          create_file(template_path)
+          fill_moonfile
+        end
+
+        private
+
+        def cwd
+          Dir.pwd
+        end
+
+        def create_project_dir
+          raise "Directory '#{@application_name}' already exists!" \
+            if Dir.exist?(project_path)
+          Dir.mkdir(project_path)
+        end
+
+        def project_path
```

Comment:
I would stick with a new directory, that's how most of these commands work (like with rails).